src/app/Caches/CacheEntries.tsx

```diff
@@ -276,7 +276,7 @@ const CacheEntries: React.FunctionComponent<any> = (props: {
           </Table>
         </CardBody>
       </Card>
-      <RecentActivityTable />
+      <RecentActivityTable cacheName={props.cacheName} />
```

Comment:
> it's a bug but not related @ryanemerson I will log a jira

Are you sure it's not related? Isn't the issue because `props` is undefined when this is executed?

Reply:
no, it isn't. you can reproduce in the latest version

Reply:
it's because the react app it's not init correctly when you come into the cache detail directly. it is not related to the change